pnpm-workspace.yaml

```diff
@@ -0,0 +1,3 @@
+packages:
+  - app
+  - presentations
```

Comment:
[issue](https://conventionalcomments.org/#labels): I don't think we're going to have multiple presentations, rather that presentation 2 is an outgrown version of presentation 1
[suggestion](https://conventionalcomments.org/#labels): let's name it `presentation`

Reply:
fixed